### ISSUE TITLE
Mailjet 3.1 API fails on mailjet.template.get()

### ISSUE THREAD

**Issue by Yenthe666** [mailjet/mailjet-apiv3-python#73]
Mailjet has an API endpoint to list all templates (ref https://dev.mailjet.com/mandrill/templates-calls/#list).
Have a look at this sample code:
```
from mailjet_rest import Client
import os
api_key = os.environ['MJ_APIKEY_PUBLIC']
api_secret = os.environ['MJ_APIKEY_PRIVATE']
mailjet = Client(auth=(api_key, api_secret))
result = mailjet.template.get()
print result.status_code
print result.json()
```
This works fine on the default V3 API, however when you set the version to `3.1` it will crash.
Replace this:
```
mailjet = Client(auth=(api_key, api_secret))
```

By this:
```
mailjet = Client(auth=(api_key, api_secret), version='v3.1')
```

Now run the code again. It now gives back a 404 response code with in the body the following:
```
b'<html>\r\n<head><title>404 Not Found</title></head>\r\n<body bgcolor="white">\r\n<center><h1>404 Not Found</h1></center>\r\n<hr><center>nginx</center>\r\n</body>\r\n</html>\r\n'
```

It looks like some callback link is invalid behind the scenes :)

**Comment by Yenthe666** [comment]
@diskovod any chance you could look at this?

**Comment by diskovod** [comment]
Sure, I'll take a look

**Comment by diskovod** [comment]
Hi, the problem here that there is no template functionality in version 3.1. Also as I can see in the documentation under the **Introduction** section you can find information about resources and sub-resources for template functionality. Once this functionality will be available under 'v3.1' wrapper also will be updated. Thanks!
